Rakefile

```diff
@@ -1,9 +1,27 @@
-# Add your own tasks in files placed in lib/tasks ending in .rake,
-# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
-
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
 
-ZIP_URL = "https://github.com/projectblacklight/blacklight-jetty/archive/v4.10.3.zip"
+BLACKLIGHT_JETTY_VERSION = '4.10.4'
+ZIP_URL = "https://github.com/projectblacklight/blacklight-jetty/archive/v#{BLACKLIGHT_JETTY_VERSION}.zip"
+
 require 'jettywrapper'
+require 'rspec/core/rake_task'
+
+RSpec::Core::RakeTask.new(:spec)
+
+desc 'Execute the test build that runs on travis'
+task ci: [:environment] do
+  if Rails.env.test?
+    Rake::Task['pulmap:setup_db_and_jetty'].invoke
+    jetty_params = Jettywrapper.load_config
+    Jettywrapper.wrap(jetty_params) do
+      Rake::Task['geoblacklight:solr:seed'].invoke
```

Comment:
rake:ci calls ::setup_db_and_jetty not ::setup. The test and dev environments *should* spin up different jetty instances.